### PR TITLE
move opengrok-tools -> tools

### PR DESCRIPTION
**Pull request by vladak**
This change renames the `opengrok-tools` directory to just `tools`.

### CODE REVIEW

**Comment by vladak** [comment]
@tulinkry can you take a look ?

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ COPY suggester/pom.xml /mvn/suggester/
 
 # distribution and opengrok-tools do not have dependencies to cache
```
Comment (tulinkry) [inline Dockerfile:17]: also only tools

File: distribution/assembly.xml [hunk 2]
```diff
@@ -20,17 +20,17 @@
         <destName>source.war</destName>
       </file>
       <file>
-        <source>${project.basedir}/../opengrok-tools/${project.build.directory}/dist/opengrok-tools-${project.python.package.version}.tar.gz</source>
+        <source>${project.basedir}/../tools/${project.build.directory}/dist/opengrok-tools-${project.python.package.version}.tar.gz</source>
```
Comment (tulinkry) [inline distribution/assembly.xml:23]: so the target archive is still opengrok-tools? I'm all for, just asking.

**Review by tulinkry — APPROVED** [review]

File: distribution/assembly.xml (hunk 2 above)
Comment (vladak) [reply distribution/assembly.xml:23]: Yes, I think it should be recognizable among all the other Python packages. 'tools' would be too generic.

File: Dockerfile (hunk 1 above)
Comment (vladak) [inline Dockerfile:17]:
```suggestion
# distribution and tools do not have dependencies to cache
```

**Comment by vladak** [comment]
Tanks @tulinkry !

**Pull request merged**